### PR TITLE
Add EURe token metadata

### PR DESCRIPTION
**Pull request by PierreLeGuen**
ERC-20: https://etherscan.io/address/0x3231cb76718cdef2155fc47b5286d82e6eda273f 
NEP-141: https://nearblocks.io/address/3231cb76718cdef2155fc47b5286d82e6eda273f.factory.bridge.near 
Aurora ERC-20: https://explorer.aurora.dev/address/0xbF7d2104C990D1e91F3e1Dc7E1418ae9AF2c8D8A

### CODE REVIEW

File: tokens/EURe.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0x3231cb76718cdef2155fc47b5286d82e6eda273f",
+    "aurora_address": "0xbf7d2104c990d1e91f3e1dc7e1418ae9af2c8d8a",
```
Comment (sept-en) [inline tokens/EURe.json:3]:
```suggestion
    "ethereum_address": "0x3231cb76718cdef2155fc47b5286d82e6eda273f",
    "near_address": "3231cb76718cdef2155fc47b5286d82e6eda273f.factory.bridge.near",
    "aurora_address": "0xbf7d2104c990d1e91f3e1dc7e1418ae9af2c8d8a",
```

**Review by sept-en — CHANGES_REQUESTED** [review]
Thanks for your PR. I added a NEP-141 address too, which is needed for consistency.

**Review by sept-en — APPROVED** [review]
The token metadata will be updated within the week.

**Comment by PierreLeGuen** [comment]
@sept-en would be good to add the NEP-141 (`near_address`) field in the `EXAMPLE_token.json`.

**Pull request merged**